examples/1-hello-world/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "hello-world",
+  "version": "0.0.1",
+  "description": "Hello world example",
+  "main": "index.js",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/copilot-extensions"
+  },
+  "scripts": {
+    "start": "node index.js",
+    "watch": "node --watch index.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@copilot-extensions/preview-sdk": "^4.0.3"
```

Comment:
I would use latest as [next.js does](https://github.com/vercel/next.js/blob/874454f3fab4cf332a8436ad38497cfc4ab91219/examples/hello-world/package.json#L9)

```suggestion
    "@copilot-extensions/preview-sdk": "latest"
```

Reply:
we could also set it to a relative path instead, as the source code can be imported directly, we don't have a build step

```suggestion
    "@copilot-extensions/preview-sdk": "../../index.js"
```